src/rebuild/srv.c

```diff
@@ -2180,7 +2186,7 @@ regenerate_task_of_type(struct ds_pool *pool, pool_comp_state_t match_states, ui
 
 /* Regenerate the rebuild tasks when changing the leader. */
 int
-ds_rebuild_regenerate_task(struct ds_pool *pool, daos_prop_t *prop)
+ds_rebuild_regenerate_task(struct ds_pool *pool, daos_prop_t *prop, uint64_t delay_sec)
```

Comment:
[Question] Looks like `delay_sec` will always be 0. Is this change still necessary?

Reply:
right, in previous version it passed in a delay for reboot, in this version i removed that change.
Did not remove the function parameter although, not sure if later can be used.